sdk/keyvault/Azure.Security.KeyVault.Administration/src/Properties/AssemblyInfo.cs

```diff
@@ -11,3 +11,4 @@
     "ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484c" +
     "f7045cc7")]
 [assembly: Azure.Core.AzureResourceProviderNamespace("Microsoft.KeyVault")]
+[assembly: Azure.Core.CodeGenSuppressTypeAttribute("AzureSecurityKeyVaultAdministrationModelFactory")]
```

Comment:
Actually, since the name was fixed, this might be fine to generate. I do appreciate you adding this similar attribute to the sharelink sample, though.

Reply:
Restored.